waspc/data/Generator/templates/server/src/jobs/_job.ts

```diff
@@ -0,0 +1,22 @@
+{{={= =}=}}
+import prisma from '../dbClient.js'
+import type { JSONValue } from '../_types/serialization.js'
+import type { Job } from './core/job.js'
+import { createJob } from './{= executorJobRelFP =}'
+{=& jobPerformFnImportStatement =}
+
+const entities = {
+  {=# entities =}
+  {= name =}: prisma.{= prismaIdentifier =},
+  {=/ entities =}
+};
+
+export type {= typeName =}<Input extends object, Output extends JSONValue | void> = Job<Input, Output, typeof entities>["jobFn"]
```

Comment:
When trying to submit a job with just a plain string as data, we get a runtime error. I also thought `Input` should be `JSONValue` since it's just saved as is in the `jsonb` field: https://github.com/timgit/pg-boss/blob/master/src/manager.js#L396

```
[Server!] error: invalid input syntax for type json
[Server!]     at /Users/ilakovac/dev/wasp/waspc/examples/crud-testing/.wasp/out/server/node_modules/pg-pool/index.js:45:11
[Server!]     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
https://github.com/brianc/node-postgres/issues/442

They suggest people to `JSON.stringify` stuff. But I propose to keep things simple and only allow `object` as input and that's it.

Reply:
But `object` is probably too permissive (field values can be functions, etc.). Why not use `JSONObject` (exported from the same file)?

Reply:
Solid suggestion, I've used `JSONObject` 👍